src/CatalogService/src/CatalogService.Api/Catalog.Api.csproj

```diff
@@ -6,6 +6,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <IncludeOpenAPIAnalyzers>true</IncludeOpenAPIAnalyzers>
     <UserSecretsId>5e78db56-d7bd-4017-a102-e7741bd27e0e</UserSecretsId>
+	<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
```

Comment:
This is actually important, and Microsoft made it explicit. What is the reason you need it? It's better to resolve the root cause, instead of swallowing it.

Reply:
Fixed.